lib/src/main/java/io/ably/lib/util/AblyError.java

```diff
@@ -0,0 +1,160 @@
+package io.ably.lib.util;
+
+public class AblyError {
+    public static final int NO_ERROR = 10000; //No error
+
+    public static final int BAD_REQUEST = 40000; // "bad request"
```

Comment:
Is this a doc comment? 
If it is, then we are good, otherwise you can add doc comment at the top 👍

Reply:
These are regular comments, if I make doc comments the whole class will lose readability as these codes won't look like list.